README.md

```diff
@@ -72,6 +94,10 @@ var obj = {
 ```
 
 -- YOUR ANSWER HERE --
+- ES5.
+- This function is anonymous.
+- obj.add (I can't remember if I need to pass the parameters like obj.add(x,y)). 
+
```

Comment:
You always need to pass arguments to  function if it defines parameters. But this is correct other than that